packages/react-scripts/template/README.md

```diff
@@ -2014,8 +2014,9 @@ For more information see [Add Firebase to your JavaScript Project](https://fireb
 
 Open your `package.json` and add a `homepage` field:
 
-```js
-  "homepage": "https://myusername.github.io/my-app",
+```json
+  "homepage": "https://myusername.github.io,         // for a GitHub user page
+  "homepage": "https://myusername.github.io/my-app", // for a GitHub project page
```

Comment:
Can we split this into two separate examples? Just a separate json snippet with text preceding it saying like "or for a GitHub project page ..."

Reply:
Sure, I'll tweak it.